--- __init__.py	2017-08-11 14:40:35.464581556 +0200
+++ __init__2.py	2017-08-11 14:40:47.677067679 +0200
@@ -655,8 +655,10 @@
             rescode = func(self_, *args, **kw)
             # If there is only a single output value, then do not expect it to
             # be iterable.
-            if len(outargs) == 1:  # rescode is not iterable
-                return rescode.__ctypes_from_outparam__()
+            if outnum == 1:  # rescode is not iterable
+                if len(outargs) == 1:
+                    rescode = rescode.__ctypes_from_outparam__()
+                return rescode
 
             rescode = list(rescode)
             for outnum, o in outargs.items():
